akka-http-core/src/main/scala/akka/http/impl/settings/ConnectionPoolSettingsImpl.scala

```diff
@@ -33,6 +36,7 @@ object ConnectionPoolSettingsImpl extends SettingsCompanion[ConnectionPoolSettin
   def fromSubConfig(root: Config, c: Config) = {
     ConnectionPoolSettingsImpl(
       c getInt "max-connections",
```

Comment:
One more thing - how does binary compatibility when I'm adding something new? Jenkins has failed because of that